easybuild/easyconfigs/v/Vampir/Vampir-8.4.1-demo-foss-2015a.eb

```diff
@@ -0,0 +1,48 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+#
+# Copyright:: Copyright 2013-2015 Juelich Supercomputing Centre, Germany
+# Authors::   Bernd Mohr <b.mohr@fz-juelich.de>
+#             Markus Geimer <m.geimer@fz-juelich.de>
+# License::   3-clause BSD
+#
+# This work is based on experiences from the UNITE project
+# http://apps.fz-juelich.de/unite/
+##
+
+easyblock='Binary'
+
+name = "Vampir"
+version = "8.4.1"
+versionsuffix = "-demo"
+variant = "linux-x86_64"
+
+homepage = 'http://www.vampir.eu'
+description = """The VAMPIR software tool provides an easy-to-use framework that enables
+ developers to quickly display and analyze arbitrary program behavior at any level of detail.
+ The tool suite implements optimized event analysis algorithms and customizable displays that
+ enable fast and interactive rendering of very complex performance monitoring data."""
+
+toolchain = {'name': 'foss', 'version': '2015a'}
+
+sources = ['vampir-%s%s-%s-setup.bin' % (version, versionsuffix, variant)]
+
+checksums = [
+    '09cf28650be36516613372c8a0d1c0d8',    # vampir-8.4.1-demo-linux-x86_64-setup.bin
```

Comment:
The demo packages are rebuilt every 2-3 month, because they are time limited. And if inbetween these rebuilds no new version was released, its the same pacakge, but with different expiration dates.